operations/helm/tests/enterprise-https-values-generated/mimir-distributed/templates/compactor/compactor-statefulset.yaml

```diff
@@ -82,7 +83,6 @@ spec:
         - name: runtime-config
           configMap:
             name: enterprise-https-values-mimir-runtime
-        
```

Comment:
Linter fails on this and when I run `make check-helm-tests` I don't get this change. Same for line 99 bellow.

Reply:
Thanks for the feedback, I just run the `make check-helm-tests`.